packages/console/src/pages/GetStarted/FreePlanNotification/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Trans, useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import FreePlanNotificationImage from '@/assets/images/free-plan-notification-image.svg';
+import PlanName from '@/components/PlanName';
+import { isCloud, isProduction } from '@/consts/env';
+import { ReservedPlanId } from '@/consts/subscriptions';
+import Button from '@/ds-components/Button';
+import useCurrentSubscription from '@/hooks/use-current-subscription';
+import { ReservedPlanName } from '@/types/subscriptions';
+
+import * as styles from './index.module.scss';
+
+function FreePlanNotification() {
+  const { data: currentSubscription, error } = useCurrentSubscription();
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.upsell.get_started' });
+  const navigate = useNavigate();
+  const isLoadingSubscription = !currentSubscription && !error;
+
+  if (
+    isProduction ||
```

Comment:
nit: better add a comment for this feature switch

Reply:
updated.